### PR TITLE
[copporch] Add safeguard during policer attribute update

### PR DESCRIPTION
**Pull request by vivekrnv**
**What I did**

Add safeguard during policer attribute update in copporch when the config is trying to change CREATE_AND_UPDATE attributes by not allowing CREATE_ONLY attributes to pass. 

If user tries to update a CREATE_ONLY attribute, it follows the current design of orchagent crash.

**Why I did it**

During fast-boot, coppmgr does a DEL folowed by SET on the diverged COPP_TABLE key, But while reading the notification, the DEL is optimized and only SET is seen and this would lead to orchagent crash even though, user is only changing CREATE_AND_UPDATE attributes.

**How I verified it**

Update the copp_cfg.j2 file to update the CIR and CBS, followed by a fast boot, This should simulate divergence in the copp config and the scenario explained above is hit and orchagent did not crash

```
<entries applied during re-conciliation>
2023-12-01.02:45:53.378712|COPP_TABLE:queue4_group3|SET|cbs:100|cir:100|meter_type:packets|mode:sr_tcm|queue:4|red_action:drop|trap_action:trap|trap_priority:4|trap_ids:dhcp,dhcpv6,lldp,udld
2023-12-01.02:45:53.378747|COPP_TABLE:queue4_group2|SET|cbs:1000|cir:600|meter_type:packets|mode:sr_tcm|queue:4|red_action:drop|trap_action:copy|trap_priority:4|trap_ids:arp_req,arp_resp,neigh_discovery
2023-12-01.02:45:53.378763|COPP_TABLE:default|SET|cbs:600|cir:600|meter_type:packets|mode:sr_tcm|queue:0|red_action:drop
2023-12-01.02:45:53.378778|COPP_TABLE:queue1_group1|SET|cbs:6000|cir:6000|meter_type:packets|mode:sr_tcm|queue:1|red_action:drop|trap_action:trap|trap_priority:1|trap_ids:ip2me
2023-12-01.02:45:53.378791|COPP_TABLE:queue4_group1|SET|cbs:6000|cir:6000|meter_type:packets|mode:sr_tcm|queue:4|red_action:drop|trap_action:trap|trap_priority:4|trap_ids:bgp,bgpv6,lacp

<New SET entry processed by copporch>
2023-12-01.02:46:37.465125|COPP_TABLE:queue4_group3|SET|cbs:500|cir:500|meter_type:packets|mode:sr_tcm|queue:4|red_action:drop|trap_action:trap|trap_priority:4|trap_ids:dhcp,dhcpv6,lldp,udld
```

Added Unit Test

**Details if related**

### CODE REVIEW

**Review by dgsudharsan — APPROVED** [review]

**Comment by dgsudharsan** [comment]
@prsunny , @StormLiangMS This change is required as a bug fix for 202305. Without this fix, lets assume we are changing some default policer field in copp_cfg.j2 (e.g. Updating CBS/CIR value as in https://github.com/sonic-net/sonic-buildimage/pull/17132) which was present in previous version (202205), then coppmgr logic will do a del of policer and then set with new fields. However due to optimization in orchagent, DEL will be ignored and the SET will set all the policer fields which includes create only fields leading to crash.
To avoid that, this PR checks if create only fields are not modified, skip it.

File: orchagent/copporch.cpp [hunk 1]
```diff
@@ -1128,6 +1150,35 @@ bool CoppOrch::trapGroupUpdatePolicer (string trap_group_name,
         for (sai_uint32_t ind = 0; ind < policer_attribs.size(); ind++)
         {
             auto policer_attr = policer_attribs[ind];
+            /*
+                Updating the CREATE_ONLY attributes of the policer will cause a crash
+                If modified, throw an error log and proceed with changeable attributes
+            */
+            if(policer_attr.id == SAI_POLICER_ATTR_METER_TYPE)
```
[the remark this answers is not in the document]
Comment (vivekrnv) [reply orchagent/copporch.cpp:1157]: coppmgr pushes all the fv's down to copporch. if we don't save the old attributes that were created, we can't log an error saying user is trying to change a CREATE_ONLY attribute.

File: orchagent/copporch.cpp [hunk 2]
```diff
@@ -1128,6 +1150,35 @@ bool CoppOrch::trapGroupUpdatePolicer (string trap_group_name,
...
+            {
+                if (policer_object.meter != (sai_meter_type_t)policer_attr.value.s32)
+                {
+                    SWSS_LOG_THROW("Trying to modify policer attribute: (meter), trap group: (%s)", trap_group_name.c_str());
```
[the remark this answers is not in the document]
Comment (vivekrnv) [reply orchagent/copporch.cpp]: Updated

**Review by prsunny — DISMISSED** [review]

**Review by dgsudharsan — APPROVED** [review]

**Comment by dgsudharsan** [comment]
@StormLiangMS  Tested on top of 202305 commit https://github.com/sonic-net/sonic-buildimage/commit/688245a724e7462765530fd8ec3a745d7372dd5e

**Pull request merged**